src/main/java/com/powsybl/openloadflow/network/LfAction.java

```diff
@@ -8,10 +8,7 @@
 package com.powsybl.openloadflow.network;
 
 import com.powsybl.action.*;
-import com.powsybl.iidm.network.Bus;
-import com.powsybl.iidm.network.Load;
-import com.powsybl.iidm.network.Network;
-import com.powsybl.iidm.network.Terminal;
+import com.powsybl.iidm.network.*;
 import com.powsybl.openloadflow.graph.GraphConnectivity;
```

Comment:
Did you remove the explicit import on purpose ?
Looks like many other files in this project don't use wilcards on import.
In case you use IntelliJ you may see  https://sergiodelamo.com/blog/intellij-idea-disable-wildcard-imports.html for a configuration that does not put wildcards on import event with more than 3 classes imported from a package.

This is cleaner in cases where the same class name can be user in different packages.

Reply:
Changed it back to what it was
Indeed, IntelliJ automatically swapped multiple imports for a wildcard import

Reply:
Luke,
  I stepped your test and it happens that when the contingency is processed a PowerShift  is created with a Variable active power. This happens in PropagatedContingency.getLoadPowerShift.
 
  I am not very familiar with load compensation, but it looks to me as a bug.The contingency on a fictive load should not add a shift of variable active power for a fictive load, in my understanding.

 So it seems that we already have duplicated code and several places to compute the active power of an IIDM Load.
 It think it would would be better if there was a single function that returned this value in the LF wordls, that would contain the  logic on fictive in this unique place. And that NO other place of OpenLoadFlow computes the variable power of a load.

Reply:
You are right, there is a problem in my implementation.

The test now reflects it.

I think I have to add the isFictitious information in the LfLostLoad object...

And then handle each lost load accordingly.

What do you think about it?

Reply:
For me the ideal solution would be that the logic that computes the variable active power of a load is in only one place.
That means for example that if in the future, you need to add a parameter to take fictive into account or not, you would have only one function to modify.
The the logic is spread in different places, for actions, for contingnencies etc... there is a maintainaility issue and a risk that if in the future that logic has to be modified it is done only in one place.

Reply:
Hi @vidaldid-rte, I have just fixed the behaviour. We have something that is working now.

Could you have a look at this final version and tell me what you think?

I agree with you that the fictitious load information is all over the place (contingency, action), but I don't see another way of doing it...
I would gladly hear your suggestions about that.

Reply:
It looks much better!!! Thank you :)
I have only one comment: don't you think it's better to name isOriginalLoadPassive instead of isIidmLoadPassive? It is coherent with the other methods' naming, and it doesn't mention a data model

Reply:
No problem for the renaming. Go ahead !